api/TimeAddressableMediaStore.yaml

```diff
@@ -1837,6 +1837,92 @@ paths:
           description: Forbidden. You do not have permission to modify this flow. It may be marked read-only.
         "404":
           description: The requested flow does not exist.
+  /objects:
+    head:
+      summary: Media Object Information
+      description: Return flow references and other information about media objects.
```

Comment:
NIT: Could you add a quick note here on how this uses a query string rather than a path var, and why? I expect it's something that'll catch people out/confuse people fairly frequently. So the more visible we can make it, the better.